api/py/ai/chronon/repo/compile.py

```diff
@@ -304,7 +304,11 @@ def _get_conf_file_path(downstream: str, downstream_class: Type[Union[Join, Grou
 
 def _print_features(obj: Union[Join, GroupBy], obj_class: Type[Union[Join, GroupBy]]) -> None:
     if obj_class is Join:
-        _print_features_names("Output Join Features", get_join_output_columns(obj))
+        features = get_join_output_columns(obj)
+        for key, value in features.items():
+            left = key.value.replace("_", " ").title()
+            right = pprint.pformat(value, width=80, compact=True)
```

Comment:
optional suggestion: `textwrap` library is also quite good for some of these formatting needs. Though I do prefer `pprint` most of the time (esp for displaying dicts).

Reply:
yes. I played around with textwrap but wasn't giving me the right output so used pprint. API seems simpler than textwrap.